nuxt.config.ts

```diff
@@ -24,7 +24,6 @@ export default defineNuxtConfig({
   robots: {
     rules: {
       UserAgent: '*',
-      Disallow: '/',
```

Comment:
Betekent dit dat ook de acceptatie omgeving geindexeerd kan worden? Is dit de bedoeling